stories/Components/UIcomponents/Accordion/accordion.scss

```diff
@@ -8,70 +8,78 @@
 
   padding: 0;
   position: relative;
-}
-
-.accordion__item {
-  border-bottom: 1px solid $color-gray-500;
-  cursor: pointer;
-  list-style: none;
-  margin-bottom: 0;
-  overflow: hidden;
-  padding: $spacing-16 0 $spacing-24;
-  position: relative;
-
-  &::before {
-    @include transition(all 0.3s ease);
-
-    background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
-    border-top-color: #f3f3f3;
-    content: '';
-    float: right;
-    height: 20px;
-    margin-left: $spacing-24;
-    width: 20px;
-  }
 
-  .accordion__heading {
+  li {
+    border-bottom: 1px solid $color-gray-500;
+    cursor: pointer;
+    list-style: none;
     margin-bottom: 0;
-  }
-
-  &.accordion--active {
-    padding-bottom: 0;
+    overflow: hidden;
+    position: relative;
+
+    button {
+      display: block;
+      width: 100%;
+      margin: 0 auto;
+      padding: $spacing-16 0 $spacing-24;
+      font-size: $font-size-25;
+      font-weight: 400;
+      line-height: 1.15;
+      border: none;
+      background: transparent;
+      text-align: left;
+
+      &::before {
+        @include transition(all 0.3s ease);
+
+        background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
+        border-top-color: #f3f3f3;
+        content: '';
+        right: $spacing-16;
+        height: 1.25rem;
+        margin-left: $spacing-24;
+        width: 1.25rem;
+        position: absolute;
+        top: $spacing-24;
+      }
 
-    .accordion__heading {
-      margin-bottom: 0.625rem;
-      padding-right: $spacing-24;
+      &.accordion--active {
+        padding-bottom: 0;
```

Comment:
@neelesh-tippe Don't remove the bottom padding for the `active` li as this causes the `accordion__panel` to jump down a bit while slide up animation(when the accordion is closing). Instead reduce the top padding for `accordion__panel`. 
Eg:
```
.accordion__panel { 
  padding: $spacing-24 $spacing-32 $spacing-32;
  @include devicebreak(medium) {
    padding: $spacing-24 $spacing-40 $spacing-40;
  }
}
```

Reply:
&.accordion--active {
        padding-bottom: 0;

 is removed from code, there were no specific use of it.